edflow/custom_logging.py

```diff
@@ -423,21 +423,63 @@ def _create_logger(name, out_dir, level=logging.INFO):
         logger = logging.getLogger(name)
         logger.setLevel(level)
 
+        logger.copyline = lambda msg: logger.log(22, msg)
+
         if not len(logger.handlers) > 0:
             ch = TqdmHandler()
             fh = logging.FileHandler(filename=os.path.join(out_dir, "log.txt"))
 
             fmt_string = "[%(levelname)s] [%(name)s]: %(message)s"
             formatter = logging.Formatter(fmt_string)
```

Comment:
is the `formatter` still needed or can we delete it?

Reply:
Yep, should be deleted.